### PR TITLE
chore: export vm.ShouldBurn

### PR DESCRIPTION
**Pull request by iand**
Required for sentinel to calculate gas outputs for each message. 

There is some noise caused by gofumpt formatting of the file.

### CODE REVIEW

File: chain/vm/vm.go
Comment (raulk) [inline chain/vm/vm.go:50]: Ideally we would introduce formatting changes in separate PRs, so that future git blames can be less noisy and more accurate. But this is OK.

**Review by raulk — APPROVED** [review]

File: chain/vm/vm.go
Comment (iand) [reply chain/vm/vm.go:50]: I'm happy to revert the formatting if you prefer

Comment (raulk) [reply chain/vm/vm.go:50]: No worries.

**Comment by iand** [comment]
Can't merge without a code owner approval.

**Comment by raulk** [comment]
Need to wait for @magik6k or @Kubuxu.

**Review by Kubuxu — APPROVED** [review]

**Pull request merged**
